### ISSUE TITLE
ui5-li: accessible-name is not behaving like aria-label

### ISSUE THREAD

**Issue by pankajpandey01** [SAP/ui5-webcomponents#5196]
### **Bug Description**
ui5-li: accessible-name is not behaving like aria-label

### **Expected Behavior**
accessible-name should behave like aria-label (https://sap.github.io/ui5-webcomponents/master/playground/advanced/accessibility/)

### **Isolated Example**
https://codesandbox.io/s/ui5-webcomponents-forked-9qpkru?file=/index.html

In this example, we have links in the list which are supposed to open in new window. So we want to announce this to screen reader user as part of accessible-name="Settings opens in new window". But JAWS announce as "Settings opens in new window settings". Here "settings" is repeated.

### **Context**
- UI5 Web Components version: {1.3.0}
- OS/Platform: {All}
- Browser: {All}
- Affected component: {ui5-li}

### **Log Output / Stack Trace / Screenshots**
![Screen Shot 2022-05-10 at 9 11 08 PM](https://user-images.githubusercontent.com/67934622/167767631-ed004ec9-6ad2-47de-9347-807e77da7258.png)

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Accessibility issue - Medium 

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: {Successfactors}

**Comment by niyap** [comment]
Hello @pankajpandey01,

The expected behaviour is to be read out the content of the list item, because the same information should be accessible both for the user that can see it and for those who can hear it. Something that is visible for the user should be accessible to the users that use screenreader as well.

Kind Regards,
Niya

**Comment by pankajpandey01** [comment]
Hi @niyap , 
Thanks for reply. Please help me with below queries.
I think content can be read by default, then what is the purpose of accessible-name?
If we want to announce some thing to non-visual user which is not needed for visual user. How we can do that?
Usually, we hare aria-description/aria-describedby for that purpose. How we can achieve that functionality in this component?

Thanks,
Pankaj

**Comment by niyap** [comment]
Hello @pankajpandey01,
The purpose of accessible-name is to add label. It works in that way, adding additional information to the text content.
Currently, aria-description is not supported due to the nature of the custom elements and their shadow DOM. 
Here, you can find more information: https://sap.github.io/ui5-webcomponents/master/playground/advanced/accessibility/

Kind Regards,
Niya

**Comment by pankajpandey01** [comment]
Hi @niyap, 
I understand that it is additional information. If content is main information and accessible-name is additional info, then order should be like first 'content' then 'accessible-name'. Currently it is reverse which is causing problem. Can we correct this order ? (If we do this order accessible-name can behave like aria-description).

Just FYI : ['aria-label' and 'aria-labelledby' takes precedence over 'content' (https://web.dev/aria-labels-and-relationships/). if they are set, only they are announced. I understand that accessible-name is not aria-abel/aria-labelledby.]

Thanks,
Pankaj

**Comment by pankajpandey01** [comment]
Hi @niyap
As per [](https://sap.github.io/ui5-webcomponents/master/playground/advanced/accessibility/,) accessibleName is mapped to aria-label. In this case, it should behave what was expected in this ticket. Can you please reopen the ticket ?
Thanks,
Pankaj

**Comment by niyap** [comment]
Hello @SAP/ui5-webcomponents-topic-p,

Could you please check whether there is an issue with the current implementation of accessibleName property?

Kind Regards,
Niya
